kafka_consumer/tests/docker/kerberos/kdc/krb5_agent.conf

```diff
@@ -0,0 +1,24 @@
+# This is a separate file since kdc = localhost:88 is necessary to access the kdc from the Agent container
+[libdefaults]
+default_realm = TEST.CONFLUENT.IO
+forwardable = true
+rdns = false
+dns_lookup_kdc   = no
+dns_lookup_realm = no
+
+[realms]
+TEST.CONFLUENT.IO = {
+	kdc = localhost:88
```

Comment:
This is the only difference between krb5.conf and krb5_agent.conf. Having `kdc = kdc` didn't allow the Agent container to see the kdc, although it was still necessary to keep it for the other containers, hence splitting this file into two for now.